test/specs/login.test.js

```diff
@@ -70,4 +70,16 @@ describe('Login Suite', () => {
       
     });
 
+    it('Deberia acceder a la pagina principal cuando se loguea con credenciales validas y luego desloguearse', async () => {
+
+      await LoginPage.abrir('login');      
+      let username = 'nsmith@demo.io'
+      let password = 'Demo123!'
+      await LoginPage.ingresarCredenciales(username, password);
+      await LoginPage.clickEnBotonSubmit()
+      await HomePage.desloguearse()
+      await expect(LoginPage.loginStateFeedback).toHaveTextContaining('Success')
+            
+    });
```

Comment:
Este test pertenece a otra suite, la de logout. Tenes dos alternativas, en mi opinion:

1. Asumis que esta suite este representado pruebas sobre el **sistema de sesion** (que incluye login y logout). En este caso, deberias reescribir el nombre de la suite a algo asi como **Session Suite**

2. Asumir que la suite solo se ocupa del login, en cuyo caso, el logout deberia estar en otra suite (otro archivo de tests, con su describe e it).

Yo preferiria la opcion 1, ya que es un solo test, y relacionado a funcionalidades de gestion de sesion, por lo que resulta razonable agruparlos